src/old_integration_test.rs

```diff
@@ -0,0 +1,271 @@
+#![allow(deprecated)]
```

Comment:
What's the purpose of keeping this test around if it's deprecated? Are we wanting to return to this after the issue is known?

Reply:
This is a valuable piece of code that only needs to be updated to current API. If we decide to include integration tests into our workflow this will be good starting point.